commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -408,12 +408,13 @@ def _prepare_segment_event_properties(in_order, total_amount, return_line_item_r
                 )
                 line_items = get_edx_items(order)
                 is_bundle = check_is_bundle(line_items)
-                event_title = ''
 
+                event_titles = []
```

Comment:
we can improve the name `event_titles` because an event usually has a single title.

what we can here is rename it to 
 ```suggestion
                refund_items_titles = []
```

Reply:
done